tools/deploy_ipk.sh

```diff
@@ -40,7 +40,13 @@ pgrep opkg | xargs kill -SIGINT
 opkg remove --force-depends prplmesh prplmesh-dwpal prplmesh-nl80211
 # currently opkg remove does not remove everything from /opt/prplmesh:
 rm -rf /opt/prplmesh
-opkg install -V2 "$DEST_FOLDER/$IPK_FILENAME"
+# rdkb platforms require --force-dependencies
+INSTALL_ARGS="-V2"
+. /etc/os-release >/dev/null 2>&1
+if [ $ID==rdk ]; then INSTALL_ARGS+=" --force-dependencies"; fi
+
+opkg install $INSTALL_ARGS "$DEST_FOLDER/$IPK_FILENAME"
```

Comment:
I guess shellcheck doesn't check this but it would have been better to keep the -V2 option here and do something like `${FORCE_DEPENDS:+--force-dependencies}` 
(It's not really a problem with the current options so it's a nitpick comment)

Reply:
done